packages/webapp/src/components/Navigation/NavBar/slideMenu/index.jsx

```diff
@@ -1,101 +1,82 @@
-import React from 'react';
+import { useState } from 'react';
+import clsx from 'clsx';
 import { ReactComponent as Logo } from '../../../../assets/images/navbar/nav-logo.svg';
 import { ReactComponent as VectorUp } from '../../../../assets/images/navbar/vector-up.svg';
 import { ReactComponent as VectorDown } from '../../../../assets/images/navbar/vector-down.svg';
 import { useTranslation } from 'react-i18next';
-import { makeStyles } from '@mui/styles';
 import { List, ListItem, ListItemText } from '@mui/material';
-import { isAdminSelector } from '../../../../containers/userFarmSlice';
-import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
 
-const useStyles = makeStyles((theme) => ({
-  container: {
-    width: '204px',
-  },
-  logo: {
-    margin: '16px 0 32px 24px',
-    cursor: 'pointer',
-  },
-  listItem: {
-    paddingLeft: theme.spacing(3),
-  },
-  ListItemText: {
-    fontSize: '16px',
-  },
-  subListItem: {
-    paddingLeft: theme.spacing(7),
-  },
-  subListItemText: {
-    fontSize: '14px',
-  },
-}));
+function PureSlideMenu({ history, closeDrawer, isAdmin, classes = {} }) {
+  const [manageOpen, setManageOpen] = useState(true);
+  const toggleManage = () => {
+    setManageOpen(!manageOpen);
+  };
 
-function SlideMenu({ history, manageOpen, closeDrawer, toggleManage, showFinances }) {
   const { t } = useTranslation();
-  const classes = useStyles();
   const handleClick = (link) => {
     history.push(link);
-    closeDrawer();
+    closeDrawer?.();
   };
-  const isAdmin = useSelector(isAdminSelector);
+
   return (
     <div role="presentation" className={classes.container}>
```

Comment:
Still uses the {classes.container} here.

Reply:
If I'm not mistaken, that's the proper way of taking styles from its parent component...?
(I noticed propType is missing, so I will add it. Thank you!)